agent/api/container/container.go

```diff
@@ -115,6 +115,8 @@ type Container struct {
 	Memory uint
 	// Links contains a list of containers to link, corresponding to docker option: --link
 	Links []string
+	//
+	LogRouter *LogRouter `json:"logRouter"`
```

Comment:
does this change need our state file version to be changed?

Reply:
Done. Changed the version to 23 and added a comment about the new change.